palace/linalg/divfree.cpp

```diff
@@ -49,6 +49,42 @@ DivFreeSolver<VecType>::DivFreeSolver(
     int print)
 {
   BlockTimer bt(Timer::DIV_FREE);
+
+  // If no boundaries on the mesh have been marked, add a single degree of freedo
```

Comment:
Thanks, @sebastiangrimberg ! My test is passing with this change. Just a small typo here. `freedo` -> `freedom`

Reply:
Thanks! Resolved in [b69eeba](https://github.com/awslabs/palace/pull/269/commits/b69eebad199ebce073173cd887c69a9d11052217).